community-meetings/2022-01-07.md

```diff
@@ -0,0 +1,19 @@
+# Agenda for the GoReleaser Community Meeting - January 7, 2022
+
+## Links for the participants
+
+- YouTube live stream /recording: [Youtube](https://www.youtube.com/channel/UCxg5N16FKrTa4Cees434pbw)
+- Current Milestones: [v1.1.0](https://github.com/goreleaser/goreleaser/milestone/3)
+
+## Topics
+
+- New Community Meetings Repo: [GoReleaser Community Meetings](https://goreleaser.com/community-meetings)
+- New Blog: [GoReleaser Blog](https://medium.com/goreleaser)
+- Ideas for first blog posts?
+- Notable Features of upcoming releases.
+- Duration of release cycle?
+- Kaniko Support [#2322](https://github.com/goreleaser/goreleaser/issues/2322)
+- add google/ko as brand new builder engine for building and pushing container images [#2556](https://github.com/goreleaser/goreleaser/issues/2556)
+
+## Other
+- ...
```

Comment:
how about: how to automate the [acceptance-tests](https://github.com/goreleaser/acceptance-tests) in GH actions, as this is done manually atm? WDYT?

Reply:
Yeah sounds good!

Reply:
done @mavogel and @caarlos0.